portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -64,7 +66,7 @@ public String getName() {
 
   // all options: "newest (descending), oldest (ascending), alphabetical, reverse-alphabetical"
   public String sort = "newest";
-  public ArrayList<String> messages = new ArrayList<String>(List.of("Hello", "Goodbye", "Thanks"));
```

Comment:
considering using SortDirection instead of creating your own type? It's not even a type, it's just a string, which could be mis-spelled and cause endless pains for debugging.